climada/hazard/base.py

```diff
@@ -760,6 +775,34 @@ def select(self, event_names=None, date=None, orig=None, reg_id=None, reset_freq
         haz.sanitize_event_ids()
         return haz
 
+    def select_tight_cent(self, buffer=0.0):
+        """
+        Reduce hazard to centroids to minimal box containing all non-zero
+        intensity points.
```

Comment:
This is a bit cryptic. Maybe 
```suggestion
        Reduce hazard to those centroids spanning a minimal box which contains all non-zero
        intensity points.
```

Reply:
Very general question: could the hazard be reduced to only those centroids that are non-zero? I.e. shape-agnostic. Since a "minimal bounding box" would still contain a ton of zero-intensity centroids, depending on the hazard path.

Reply:
One could, but this is not necessarily a good idea. Because then an exposure point might be assigned a centroid with non-zero intensity, although it is closer to a (now removed) centroid with zero intensity.

Reply:
Now that I think of it, this means the default buffer should be equal to the default 'assign_centroids' threshold.